static/app/components/replays/replayCurrentLocation.tsx

```diff
@@ -0,0 +1,38 @@
+import {Flex} from '@sentry/scraps/layout/flex';
+import {Tooltip} from '@sentry/scraps/tooltip/tooltip';
+
+import ErrorBoundary from 'sentry/components/errorBoundary';
+import ReplayCurrentLocationInput from 'sentry/components/replays/replayCurrentLocationInput';
+import {IconFatal} from 'sentry/icons/iconFatal';
+import {useReplayReader} from 'sentry/utils/replays/playback/providers/replayReaderProvider';
+import BrowserOSIcons from 'sentry/views/replays/detail/browserOSIcons';
+import ReplayViewScale from 'sentry/views/replays/detail/replayViewScale';
+
+interface Props {
+  isLoading: boolean;
+}
+
+export default function ReplayCurrentLocation({isLoading}: Props) {
+  const replay = useReplayReader();
+  const isVideoReplay = replay?.isVideoReplay();
+
+  return (
+    <Flex align="center" flex="1" gap="sm" justify="between" radius="md">
```

Comment:
Pretty minor but there's a small spacing difference:

Before:

<img width="136" height="45" alt="Image" src="https://github.com/user-attachments/assets/acb679c9-d726-4c98-936e-550f62b75011" />

After:

<img width="135" height="48" alt="Image" src="https://github.com/user-attachments/assets/63b9e874-d30b-4edd-a833-c6fdd78a3506" />

Changing `gap="sm"` to `gap="lg"` gets it back to the way it was before
```suggestion
    <Flex align="center" flex="1" gap="lg" justify="between" radius="md">
```